member/src/main/scala/com/improving/app/member/Main.scala

```diff
@@ -1,29 +1,58 @@
 package com.improving.app.member
 
-import com.improving.app.member.domain.Member
-import kalix.scalasdk.Kalix
-import org.slf4j.LoggerFactory
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
+import com.improving.app.member.api.{MemberServiceHandler, MemberServiceImpl}
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.StrictLogging
+
+import scala.concurrent.duration.DurationInt
+import scala.concurrent.{ExecutionContext, Future}
+import scala.language.postfixOps
+import scala.util.{Failure, Success}
 
 // This class was initially generated based on the .proto definition by Kalix tooling.
 //
 // As long as this file exists it will not be overwritten: you can maintain it yourself,
 // or delete it so it is regenerated as needed.
 
```

Comment:
You are deleting the kalix code in your akka implementation. I don't think the above kalix generated comment is applicable any more.

Reply:
Removed